.envs/.django

```diff
@@ -4,15 +4,15 @@ USE_DOCKER=yes
 IPYTHONDIR=/app/.ipython
 # Redis
 # ------------------------------------------------------------------------------
-REDIS_URL=redis://redis:6379/0
+REDIS_URL="redis://127.0.0.1:6379/0"
```

Comment:
dont touch this file, it might break dev

We can use `READ_DOT_ENV_FILE` [var](https://github.com/SCOAP3/scoap3/blob/main/config/settings/base.py#L15 ) to disable variable load from this file and pass differently the vars needed (e.g. environment, test settings - might break test locally - , etc)